### PR TITLE
added missing dependencies fixes #396

### LINKED ISSUE #396: Add missing dependencies to packages
There are some missing dependecies in some of our packages. We need to add them to package.json and the tsconfig.json if it is our package.

## Missing dependencies

🌀 needs to be added to tsconfig.json
📦 needs to be added to package.json

- [x] @sensenet/authentication-jwt
  - [x] tslib 📦
- [x] @sensenet/authentication-google
  - [x] tslib 📦
- [x] @sensenet/client-core
  - [x] tslib 📦
- [x] @sensenet/client-utils
  - [x] tslib 📦
- [x] @sensenet/control-mapper
  - [x] tslib 📦
- [x] @sensenet/controls-react
  - [x] tslib 📦
- [x] @sensenet/default-content-types
  - [x] tslib 📦
- [x] @sensenet/document-viewer-react
  - [x] @furystack/inject 📦
  - [x] tslib 📦
- [x] @sensenet/hooks-react
  - [x] @furystack/inject 📦
  - [x] @furystack/logging 📦
  - [x] semaphore-async-await 📦
  - [x] tslib 📦
- [x] @sensenet/icons-react
  - [x] tslib 📦
- [x] @sensenet/list-controls-react
  - [x] tslib 📦
- [x] @sensenet/pickers-react
  - [x] @sensenet/client-utils 🌀📦
  - [x] tslib 📦
- [x] @sensenet/query
  - [x] tslib 📦
- [x] @sensenet/redux-promise-middleware
  - [x] tslib 📦
- [x] @sensenet/redux
  - [x] tslib 📦
- [x] @sensenet/repository-events
  - [x] @sensenet/default-content-types 🌀📦
  - [x] tslib 📦
- [x] @sensenet/search-react
  - [x] @sensenet/client-utils 🌀📦
  - [x] tslib 📦

## Other tasks
- [x] remove tslib from the root package.json

### PR DESCRIPTION
**Pull request by sajarindider**
@B3zo0 I'd love your review on this, let me know if it's up to your standards.

### CODE REVIEW

File: packages/sn-client-auth-google/package.json [hunk 1]
```diff
@@ -37,7 +37,9 @@
   "homepage": "https://sensenet.com",
   "dependencies": {
     "@sensenet/authentication-jwt": "^1.0.16",
-    "@sensenet/client-utils": "^1.6.5"
+    "@sensenet/client-utils": "^1.6.5",
+    "tslib": "^1.9.3"
+
```
Comment (B3zo0) [inline packages/sn-client-auth-google/package.json]:
```suggestion
```

File: packages/sn-search-react/package.json [hunk 2]
```diff
@@ -33,10 +33,13 @@
...
     "@sensenet/icons-react": "^1.2.13",
     "@sensenet/query": "^1.1.9",
+    "@sensenet/client-utils": "^1.6.5",
     "autosuggest-highlight": "^3.1.1",
     "react": "^16.9.0",
     "react-autosuggest": "^9.4.3",
-    "react-dom": "^16.9.0"
+    "react-dom": "^16.9.0",
+    "tslib": "^1.9.3"
+
```
Comment (B3zo0) [inline packages/sn-search-react/package.json]:
```suggestion
```

**Comment by B3zo0** [comment]
Also, please change the branch to develop.

**Comment by sajarindider** [comment]
I made a bit of a mess with this one (although i learned a lot as a result). I think it'd be cleaner if i just opened up another pull request. Sorry about this!

**Pull request closed without merging**